webportal/lambda/app.py

```diff
@@ -1529,8 +1529,10 @@ def get_instances_prices(cpu, memory, os):
         terms = json.loads(data1)
         pricePerUnit=(round(float(terms['pricePerUnit']['USD']),4))
         info = {"instanceFamily" : instanceFamily,"instanceType" : instanceType,"operatingSystem" : operatingSystem,"vcpu" : VCPU, "memory" : MEMORY,"storage" : storage, "cost" : pricePerUnit}
+        print(info)
```

Comment:
Should we be printing this?

Reply:
I removed it. The only data being printed is public AWS EC2 information so there is no security problem here. Just an imperfection I suppose.